WooCommerce/Classes/ViewRelated/Orders/Order Details/Product List Section/Fulfillment/FulfillViewController.xib

```diff
@@ -24,7 +22,7 @@
             <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMaxY="YES"/>
             <subviews>
                 <tableView clipsSubviews="YES" contentMode="scaleToFill" alwaysBounceVertical="YES" style="grouped" separatorStyle="default" rowHeight="-1" estimatedRowHeight="-1" sectionHeaderHeight="-1" estimatedSectionHeaderHeight="44" sectionFooterHeight="1" translatesAutoresizingMaskIntoConstraints="NO" id="TsO-Qh-HeU">
-                    <rect key="frame" x="0.0" y="64" width="375" height="603"/>
+                    <rect key="frame" x="0.0" y="44" width="375" height="623"/>
```

Comment:
I don't know if this is a wanted change, or an automatic modification of Xcode.

Reply:
I did make some changes to this xib, updating the `UIButton` style from `system` to `custom` 👇 in line 50 - no idea why these rect diffs were generated though, it's one of the drawbacks of using xib 😐

Reply:
This is auto-generated from Xcode when you open the xib. It's just updating the Y position for the tableview based on the device you have selected in interface builder. [More details here](https://stackoverflow.com/questions/45780312/added-device-tag-in-storyboard-file-with-xcode-8-3).